packages/manager/modules/pci/src/projects/project/storages/databases/database/users/add/add.constants.js

```diff
@@ -1,6 +1,7 @@
 export const ADD_USER_FORM_RULES = {
   name: {
-    pattern: /^[a-zA-Z0-9_][a-zA-Z0-9_.-]{0,31}$/,
+    pattern: /^[a-zA-Z0-9_][a-zA-Z0-9_.-]{0,}$/,
```

Comment:
For simplicity
```suggestion
    pattern: /^\w[\w.-]*$/,
```

Reply:
Changed here: https://github.com/ovh/manager/pull/6263/commits/333eeca5e16d628aaf730ba49688c30041b03112